ScarletEditor/Sources/PropertyEditor/Private/PropertyEditor.cpp

```diff
@@ -3,6 +3,7 @@
 #include "Editor.h"
 #include "ECS.h"
 #include "Widgets.h"
+#include "Components/RigidBodyComponent.h"
```

Comment:
shouldn't you include Achilles.h ?

Reply:
I don't need all of Achilles, just the rigidbody component. In principal you should be as specific as possible when including.